### PR TITLE
Edit user

### PR DESCRIPTION
**Pull request by cpandya231**

### CODE REVIEW

**Comment by cpandya231** [comment]
[Implement Edit User feature](https://trello.com/c/3Du5lcum/26-implement-edit-user-feature)

File: src/app/users/create-user/create-user.component.ts [hunk 1]
```diff
@@ -1,56 +1,137 @@
...
   FormData = MasterForms.CREATE_USER_FORM_TEMPLATE;
   constructor(private userService: UsersService,
     private roleService: RoleService,
     private departMentService: DepartMentService,
     private router: Router,
-  ) { }
+    private activatedRoute: ActivatedRoute,
+  ) {
+    const navigation: any = this.router.getCurrentNavigation();
+    if (navigation.extras.state) {
```
Comment (akashP2312) [inline src/app/users/create-user/create-user.component.ts:38]: This won't work if we refresh the page. We won't have any data on navigation state then.

**Review by akashP2312 — CHANGES_REQUESTED** [review]
change requested

File: src/app/users/create-user/create-user.component.ts (hunk 1 above)
Comment (cpandya231) [reply src/app/users/create-user/create-user.component.ts:38]: Correct, but in that case I am fetching data using apis, check on line:58

**Review by akashP2312 — APPROVED** [review]
OKay

**Pull request merged**